ui/perfherder/alerts/AlertTableRow.jsx

```diff
@@ -430,7 +430,7 @@ export default class AlertTableRow extends React.Component {
             this.getTitleText(alert, alertStatus)
           )}
         </td>
-        <td className="table-width-lg">
+        <td className="table-width-lg information-container">
```

Comment:
Platform cell looks like it has a double line on the upper side.
We can add the `detail-hint` class to give the icon a dotted underline when the user hovers on over it.